yellowbrick/gridsearch/pcolor.py

```diff
@@ -0,0 +1,225 @@
+        GridSearchCV object to be fit.
+
+    y  : ndarray or Series of length n or None (default None)
+        An array or series of target or class values.
+
+    ax : matplotlib axes
+        The axes to plot the figure on.
+
+    classes : list of strings
+        The names of the classes in the target
+
+    Returns
+    -------
+    ax : matplotlib axes
+        Returns the axes that the classification report was drawn on.
+    """
+    # Instantiate the visualizer
+    visualizer = GridSearchColorPlot(model, param_1, param_2, ax=ax, **kwargs)
+
+    # Fit if necessary
+    if X is not None:
+        visualizer.fit(X, y)
+    else:
+        visualizer.draw()
+
+    # Return the axes object on the visualizer
+    return visualizer.ax
+
+
+class GridSearchColorPlot(GridSearchVisualizer):
+    """
+    Create a color plot showing the best grid search scores across two
+    parameters.
+
+    Parameters
+    ----------
+    model : Scikit-Learn grid search object
+        Should be an instance of GridSearchCV. If not, an exception is raised.
+
+    param_1 : string
+        The name of the parameter to be visualized on the horizontal axis.
+
+    param_2 : string
+        The name of the parameter to be visualized on the vertical axis.
+
+    ax : matplotlib Axes, default: None
+        The axes to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+
+    colormap : string or cmap, default: 'RdBu_r'
+        optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+    >>> from yellowbrick.gridsearch import GridSearchColorPlot
+    >>> from sklearn.model_selection import GridSearchCV
+    >>> from sklearn.svm import SVC
+    >>> gridsearch = GridSearchCV(SVC(),
+                                  {'kernel': ['rbf', 'linear'], 'C': [1, 10]})
+    >>> model = GridSearchColorPlot(gridsearch, param_1='kernel', param_2='C')
+    >>> model.fit(X)
+    >>> model.poof()
+    """
+
+    def __init__(self, model, param_1, param_2, colormap='RdBu_r', ax=None,
+                 **kwargs):
+        # super(GridSearchColorPlot, self).__init__(model, ax=ax, **kwargs)
+        super().__init__(model, ax=ax, **kwargs)
+        self.param_1 = param_1
+        self.param_2 = param_2
+        self.colormap = colormap
+
+    def draw(self):
+        # Project the grid search results to 2 dimensions
+        x_vals, y_vals, best_scores = param_projection(
+            self.cv_results_, self.param_1, self.param_2
+        )
+
+        # Mask nans so that they can be filled with a hatch
+        data = np.ma.masked_invalid(best_scores)
```

Comment:
I like this a lot! I'm going to have to use this in other things! The hatch nans thing I mean.